components/camel-jetty-common/src/main/java/org/apache/camel/component/jetty/JettyHttpComponent.java

```diff
@@ -134,6 +134,7 @@
     protected boolean useXForwardedForHeader;
     private Integer proxyPort;
     private boolean sendServerVersion = true;
+    private QueuedThreadPool _queuedThreadPool;
```

Comment:
Can we use the same approach for variable name? just queuedThreadPool instead of _queuedThreadpool? Thanks.

Reply:
done